README.md

```diff
@@ -2,24 +2,26 @@
 
 Test server for executing FHIR-based Electronic Clinical Quality Measures (eCQMs).
 
-- [Installation](#installation)
-
-  - [Prerequisites](#prerequisites)
-  - [Local Installation](#local-installation)
-  - [Testing](#testing)
-  - [MongoDB](#mongodb)
-  - [Redis Installation](#redis-installation)
-  - [Docker](#docker)
-
-- [Usage](#usage)
-
-  - [Database Setup](#database-setup)
-  - [CRUD Operations](#crud-operations)
-  - [Searches](#searches)
-  - [Supported FHIR Operations](#supported-fhir-operations)
-  - [Bulk Import](#bulk-import)
-
-- [License](#license)
+- [Data Exchange for Quality Measures (DEQM) Test Server](#data-exchange-for-quality-measures-deqm-test-server)
+  - [Installation](#installation)
+    - [Prerequisites](#prerequisites)
+    - [Local Installation](#local-installation)
+    - [Testing](#testing)
+    - [MongoDB](#mongodb)
+    - [Redis Installation](#redis-installation)
+    - [Docker](#docker)
+  - [Usage](#usage)
```

Comment:
Is there a reason you removed the whitespace between Docker, Usage, Bulk Import, License? If not, I would maybe add that back since it just looks nicer :)